licenses/LICENSE-bootstrap.txt

```diff
@@ -1,13 +1,22 @@
-Copyright 2013 Twitter, Inc.
+The MIT License (MIT)
```

Comment:
We need to update `LICENSE-binary` as well

Reply:
I just updated the license-binary files associated with bootstrap and datatables.

Reply:
We should also update here:
https://github.com/apache/spark/blob/master/LICENSE-binary#L419